libs/books/feature/src/lib/reading-list/reading-list.component.scss

```diff
@@ -40,6 +40,13 @@
   white-space: nowrap;
 }
 
+.mat-icon-button-gray {
+  color: gray;
```

Comment:
We need to use colour value from `variables.scss` file.
Same is applicable for other class also.

Reply:
Done